telegram/ext/messagehandler.py

```diff
@@ -51,6 +51,11 @@ class MessageHandler(Handler):
             ``chat_data`` will be passed to the callback function. It will be a ``dict`` you
             can use to keep any data related to the chat that the update was sent in.
             For each update in the same chat, it will be the same ``dict``. Default is ``False``.
+        message_updates (Optional[bool]): Should "normal" message updates be handled? Default is
```

Comment:
I wonder if we could use a filter for this instead? Now that it is actually possible to use more than one filter. Or is it more clear/explicit and nice using this method?